### DISCUSSION TITLE
The Data Object Data Pool option cannot be used. once the development server had been moved from the sandbox server.

### DISCUSSION THREAD

**Discussion by arulselvan008** [pimcore/pimcore#15549] [Q&A / Support] [answered]
The admin page is functioning properly after I moved my code base and database to a new server, but I am **unable to use the portal engine pages** because I keep getting the following error: **Warning: Array to string conversion** in ListableFieldFormatter.php file
![image](https://github.com/pimcore/pimcore/assets/106297450/ad93ef5b-84a5-4abf-af94-f4bf65a9772b)

I made an effort to fix this, but I couldn't. My sandbox and UAT response were added for your reference.

Data pool in the sandbox:
![image](https://github.com/pimcore/pimcore/assets/106297450/5927f2c8-d0de-4fcf-bda1-d63a0f60bcb5)

Data pool in UAT:
![image](https://github.com/pimcore/pimcore/assets/106297450/cea71c2c-6b85-4a9d-90e9-d78d248d51dd)
![image](https://github.com/pimcore/pimcore/assets/106297450/610f72d8-d1ff-4c8a-a775-d761192ee597)

While the details page functions properly, the Asset Data Pool also exhibits thumbnail errors in UAT.
![Screenshot 2023-07-18 162514](https://github.com/pimcore/pimcore/assets/106297450/53e3deb2-dd27-4e25-901d-b5a27611f0a5)

Kindly help us achieve this.

Please give me any instructions you may have on how to transfer or copy the project sandbox server to a new development server. the entire undertaking, code alterations, and database?

**Comment by arulselvan008** [comment]
@fashxp 
anybody Can you help us here?

**Reply by kingjia90** [reply, marked as answer]
Hi @arulselvan008 ,
good news, we fixed it and will be tagged in  `3.1.2` release next week
If you have access to the portal engine repository, this is the PR https://github.com/pimcore/portal-engine/pull/442

Thank you again for reporting!

**Reply by arulselvan008** [reply]
Hi @kingjia90 ,

Very Good Morning , Thank you for the update.
